src/utils/helpers.js

```diff
@@ -12,21 +12,21 @@ export function getUKFormattedDate(dateString, options) {
   return Intl.DateTimeFormat('en-GB', options).format(dateObj);
 }
 
+export function getCurrentMonth () {
+  return getUKFormattedDate(Date.now(), { year: 'numeric', month: 'long' });
+}
+
 export function getUniqueMonths(expenseArray) {
-  return expenseArray.reduce((acc, expense) => {
+  return expenseArray.reduce((uniqueMonths, expense) => {
     let formattedExpenseDate = getUKFormattedDate(new Date(expense.date), {
-      year: '2-digit',
+      year: 'numeric',
       month: 'long',
     });
-
-    if (
-      acc.lentgh !== 0 &&
-      formattedExpenseDate &&
-      !acc.includes(formattedExpenseDate)
-    ) {
-      acc.push(formattedExpenseDate);
+    
+    if ( formattedExpenseDate && !uniqueMonths.includes(formattedExpenseDate)) {
+      uniqueMonths.push(formattedExpenseDate);
     }
+    return uniqueMonths;
 
-    return acc;
-  }, []);
+  }, [getCurrentMonth()]);
```

Comment:
`months` will always include the `currentMonth` even without any `expenses` nor `filteredExpenses` for that month